 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6f0ee03..abe8377 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -111,7 +111,7 @@ Install_basic_package_files(${PROJECT_NAME}
                             COMPATIBILITY AnyNewerVersion
                             VARS_PREFIX ${PROJECT_NAME}
                             NO_CHECK_REQUIRED_COMPONENTS_MACRO
-                            DEPENDENCIES "YARP" "gz-sim${GZ_SIM_VER}")
+                            DEPENDENCIES "YARP COMPONENTS robotinterface os" "gz-sim${GZ_SIM_VER}")
 
 # Add the uninstall target
 include(AddUninstallTarget)
